README.md

```diff
@@ -61,6 +61,7 @@ or
 | networkInstrumentation | Bool | Enable span creation for network activities | true |
 | enableDiskCache | Bool | Enable disk caching of exported spans. All spans will be written to disk and deleted on a successful export. | false |
 | spanDiskCacheMaxSize | Int64 | Threshold in bytes from which spans will start to be dropped from the disk cache (oldest first). Only applicable when disk caching is enabled. | 25 MB |
+| sessionBaseSamplingRatio | Double | Percentage of sessions to sample.  Expressed as 0.0 - 1.0. | 1.0 |
```

Comment:
What is the word "base" doing in this (publicly documented) name?  Can't this just be `sessionSamplingRatio`?